polyfills/Object.prototype.toString/polyfill.js

```diff
@@ -0,0 +1,6 @@
+var Object_prototype_toString = Object.prototype.toString
+Object.prototype.toString = function() {
+    if (this === null) return "[object Null]";
```

Comment:
For reference, it seems this was discussed somewhere else and came to the same conclusion - https://github.com/es-shims/es5-shim/issues/304#issuecomment-104705988